src/SonarQube.Client/Services/SonarQubeClient.cs

```diff
@@ -159,7 +176,7 @@ public Task<Result<PropertyResponse[]>> GetPropertiesAsync(ConnectionRequest con
             var query = AppendQueryString(QualityProfileExportAPI, "?name={0}&language={1}&exporterKey={2}",
                 request.QualityProfileName, request.LanguageKey, roslynExporterName); // TODO: should handle optional params
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this 'TODO' comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1135)